Cargo.lock

```diff
@@ -28,9 +28,9 @@ checksum = "98161a4e3e2184da77bb14f02184cdd111e83bbbcc9979dfee3c44b9a85f5602"
 
 [[package]]
 name = "async-trait"
-version = "0.1.57"
+version = "0.1.58"
```

Comment:
Is it intentional / important that so much of Cargo.lock changed here?

Reply:
This was sorta a byproduct of me needing to update + clean to get steno to build locally, but I can revert it if you want to keep the changes out of this PR.

Reply:
Reverted the updates to the lockfile